index.js

```diff
@@ -1,13 +1,16 @@
 /* eslint-disable max-classes-per-file */
```

Comment:
- [ ] Great job working on this milestone. You are not suppose to disable eslint, rather fix the errors resulting from running the linter checks.

Reply:
hie @t-yanick  What we did was advised in the project requirements hence we are a little bit confused, could you please shed some more light on how to do that?